ci-operator/step-registry/mtr/deploy-windup/mtr-deploy-windup-ref.yaml

```diff
@@ -0,0 +1,17 @@
+ref:
+  as: mtr-deploy-windup
+  from: cli
+  commands: mtr-deploy-windup-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  env:
+  - name: WINDUP_NAMESPACE
+    documentation: Namespace to deploy Windup in.
+    default: mtr
+  - name: WINDUP_VOLUME_CAP
+    documentation: Windup volume capacity.
+    default: 5Gi
+  documentation: |-
+    Deploy Windup to the $WINDUP_NAMESPACE (default "mtr") namespace with $WINDUP_VOLUME_CAP (default 5Gi)volumeCapacity and wait 5 minutes for it to finish deploying.
```

Comment:
I have noticed most of `yaml`, `json`, and other files missing newline at EOF. I hope its not an issue, but it should be consistent

Reply:
I can go through and add them!